ignite/distributed/comp_models/base.py

```diff
@@ -172,7 +178,7 @@ def _apply_op(
                 out_dtype = tensor.dtype
                 tensor = tensor.to(self._collective_op_dtype)
 
-        tensor = fn(tensor, *args, **kwargs)
+        tensor = fn(tensor, group, *args, **kwargs)
```

Comment:
`def _do_all_reduce` takes tensor, op and group. So it occurs error like here.
Should the position of group and op be changed or is there other method?
https://github.com/pytorch/ignite/actions/runs/3043053147/jobs/4901893833#step:9:213

Reply:
Where is `op` in this call ? Can't you hide group in kwargs like op ?